Proposals/BTC Forks Roadmap Proposal.md

```diff
@@ -1,81 +1,44 @@
 
-Our mission is simple: to advance Bitcoin scaling while making a powerful statement on governance by executing a successful hardfork spinoff of the Bitcoin network and facilitating a process that allow further spinoffs in the future..
+Our initial mission is simple: to advance Bitcoin scaling while making a powerful statement on governance by executing a successful hardfork spinoff of the Bitcoin network and facilitating a process that allow further spinoffs in the future.
 
 The Bitcoin experiment has not failed. It is only just getting started!!
 
----
-
-###3. Goals of the Project
-
-1. Give users and market freedom of choice.
-2. Allow bitcoin to scale.
-3. Improve consensus mechanisms.
-4. Improve security of code produced.
-5. Improve network decentralisation.
-6. Improve development productivity.
-
----
-
-###4. Rough Schedule
 
-**Months 1-3 :** 
-Community development, establishment of necessary channels, build, and testing infrastructure, and community release of several proposed forks (including multiple client bases and PoW implementations). Our goal is to create a collaborative environment in which forks can be proposed, tested, evaluated, and deployed by the community and marketplace. We will research key challenges, like PoW changes, difficulty adjustment, script limits, and effects of blocksize cap through tests on our public test network.
+##3. Statement of Need
 
-**Months 3-6 :** 
-Client release of final binaries, and establishment of exchange relationships necessary to begin trading of dominant fork. Deployment of required network infrastructures, including an initial node and mining network. We will focus on getting a fork ready for widespread use, getting a working product to the marketplace.
+*Describes the problem.*
 
-**Months 6+ :**
-Support of the initial and any further forks through the creation of community resources for comparison, discussion, and decision making. We will faciliatate the overall Bitcoin governance process by encouraging sane forks, regardless of whether or not they are contentious. We will aim to provide the market with all resources and information necessary to make informed decisions.
+**Summary**
 
----
+The aim of bitcoin is, or should be, to become the number one transaction network in the world. It will free up the worlds capital and offer a new era of economic growth and freedom. In the 7 years since bitcoin was released by Satoshi Nakamoto in 2009 it has grown massively in scale to a market cap of over $10b and handles around 2.5 transactions every second. Although, this is very impressive for a completely new technology, in comparison to other financial assets this is still very small. For example the market cap of gold is ~$8t and the market cap of Apple ~$580b.
```

Comment:
People will be able to instantly and quickly trade all sorts of different assets globally without having to go through numerous authorities and red tape.

That's the theory anyway.